### ISSUE TITLE
Can't insert data - bad timestamp

### ISSUE THREAD

**Issue by sergeda** [fsanaulla/chronicler#61]
I can't insert data.
I've tried to create point like this:
```
val point = Point(measurement = "table", time = System.nanoTime()).addTag("key", data.key)
val point2 = Point(measurement = "table", time = currentTimeMillis()).addTag("key", data.key)
    database.writePoint(
      point,
      precision = Precisions.HOURS,
      consistency = Consistencies.ALL
    )

    database.writePoint(
      point2,
      precision = Precisions.HOURS,
      consistency = Consistencies.ALL
    )
```
I also tried to not specifying timestamp. Still didn't work.
Can you add example how to use writePoint method to the docs?

**Comment by fsanaulla** [comment]
Hello @sergeda, thank for your question. Your problem is in your time format.
Take look on [it](https://docs.influxdata.com/influxdb/v1.5/write_protocols/line_protocol_reference/#description-of-syntax). Influx handle timestamp in nano precision.
To solve your porblem:
```
val nanoTime = System.currentTimeMillis() * 1000000 // nanos precision
val point = Point(measurement = "table", time = nanoTime)
   .addTag("key", data.key)
```
Otherwise, you can not specify the time at all, and it will be generated on the server. You can check it by simple `SELECT`.
I will update docs, to mention it.

**Comment by sergeda** [comment]
@fsanaulla Actually in my example I have shown that I've tried noontimes as well. But I've tried your code as well. No luck. Still getting that error about incorrect timestamp.

**Comment by sergeda** [comment]
After some playing I believe it's problem with my database or my data. Will investigate later.

**Comment by sergeda** [comment]
@fsanaulla I have found what cause of the problem is. It happens when I'm trying to save multiple values. Then client send data in this format: 
`'test,tag=0029203708 ttt=5.6 ggg=5.8 1528654167009000000'`
and server responds with: 

> {"error":"unable to parse 'test,tag=0029203708 ttt=5.6 ggg=5.8 1528654167009000000': bad timestamp"}

But the real cause of the problem is not a timestamp but missing comma between values. I recreated it using curl and request

> curl -i -XPOST 'http://localhost:8086/write?db=test' --data-binary 'test,tag=0029203708 ttt=5.6,ggg=5.8 1528654167009000000'

works just fine
